bin/librato-storm-kafka

```diff
@@ -236,13 +236,18 @@ def do_thr()
     break if work.nil?
 
     begin
-      work = monitor_partition(work)
+      result = monitor_partition(work)
     rescue Errno::ETIMEDOUT, Errno::EHOSTUNREACH => e
```

Comment:
+1 to HB. my suggestion about catching all exceptions was just because we'd not stop script processing when some new or unexpected error comes up, and the HB/PD notification could include the exception class name as well.